content/en/docs/setup/additional-setup/external-istiod/index.md

```diff
@@ -0,0 +1,309 @@
+---
+title: Installing External Istiod [experimental]
+description: Install external Istiod and remote config cluster.
+weight: 46
+keywords: [external,istiod,remote,config]
+aliases:
+    - /docs/setup/kubernetes/additional-setup/external-istiod/
+owner: istio/wg-environments-maintainers
+test: no
+---
+
+{{< warning >}}
+This feature is actively in [development](https://github.com/istio/community/blob/master/FEATURE-LIFECYCLE.md) and is
+considered `pre-alpha`.
+{{< /warning >}}
+
+## Introduction
+
+The {{< gloss >}}external control plane{{< /gloss >}} deployment model enables mesh operators to install and manage mesh control planes on separate external clusters. This deployment model allows a clear separation between mesh operators and mesh admins. Istio mesh operators can run Istio control planes for mesh admins while mesh admins control the configuration of the control plane without worrying about installing or managing it.
```

Comment:
I think the wording of the first sentence should be something like:

```
This guide walks you through the installation of an external control plane. In
this deployment model, the mesh operator configures the mesh to use a
control plane residing outside the mesh. An external control plane can reside in
another cluster or be managed by a cloud vendor.
```

Reply:
Also, please confirm that we have this content somewhere in our deployment models guide.

Reply:
Thanks!  I 've added the first line you had. I think the other messages are similar to what I have.  Yes this is also covered in deployment guide like 
```A cloud vendor’s managed control plane is a typical example of an external control plane.``` 
```Instead of running control planes in primary clusters inside the mesh, a service mesh composed entirely of remote clusters can be controlled by an external control plane. This provides isolated management and complete separation of the control plane deployment from the data plane services that comprise the mesh.```